server/controllers/request.js

```diff
@@ -0,0 +1,97 @@
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+const Request = require("../models/Request");
+
+// @route GET /request
+// @desc gets all requests for loged-in dog sitter
+// @access Private
+exports.getRequests = asyncHandler(async (req, res, next) => {
+    const { id: sitterId } = req.user; 
+    
+    const requests = await Request.find({ sitterId })
```

Comment:
A user may either be a sitter or an owner. Why don't we query both fields here?

Eg: `$or: [{ user: req.user.id }, { sitter: req.user.id }]`

Reply:
Oh, ok. I was under the impression that only dog sitters could get to this endpoint. I understand it better now, I'll make the change.